ajax/libs/lawnchair/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lawnchair",
+  "license": "MIT",
+  "filename": "lawnchair.min.js",
+  "keywords": [
+    "lawnchair",
+    "lawnchairjs",
+    "lawnchair.js",
+    "Gruntfile",
+    "Gruntfile.js",
+    "Grunfilejs"
```

Comment:
Grunfilejs would be meaningless here, it's a tool to control the build process of javascript that this lib uses it, besides this problem, your 6 keywords I would say it's only 2 are valid, others are duplicated form of them.